sycl/include/sycl/ext/oneapi/experimental/graph.hpp

```diff
@@ -340,7 +340,14 @@ class __SYCL_EXPORT modifiable_command_graph
   /// @param path The path to write the DOT file to.
   /// @param verbose If true, print additional information about the nodes such
   /// as kernel args or memory access where applicable.
+#ifdef ___INTEL_PREVIEW_BREAKING_CHANGES
+  void modifiable_command_graph::print_graph(const std::string path,
+                                             bool verbose) const {
```

Comment:
Is there a reason that `verbose` no longer defaults to `false` when breaking changes are enabled? This seems to have changed from the previous PRs behaviour.

Reply:
Argh, should be fixed now. Thanks!